spec/frequency_table_spec.rb

```diff
@@ -7,109 +7,88 @@
   include_context "with tables for holdings"
 
   let(:ft) { described_class.new }
-  let(:ht1) {
-    build(
-      :ht_item,
-      :spm,
-      ocns: [5],
-      access: "deny",
-      rights: "ic",
-      collection_code: "MIU"
-    )
-  }
-  let(:ht2) {
-    build(
-      :ht_item,
-      :spm,
-      ocns: [5, 6],
-      access: "deny",
-      rights: "ic",
-      collection_code: "PU"
-    )
-  }
-
-  before(:each) do
-    Cluster.each(&:delete)
-    Cluster.create(ocns: ht2.ocns)
-    insert_htitem ht1
-    insert_htitem ht2
-  end
+  let(:umich_data) { {umich: {spm: {"1": 1}}} }
+  let(:upenn_data) { {upenn: {spm: {"1": 1}}} }
+  let(:ft_data) { umich_data.merge upenn_data }
+  let(:ft_with_data) { described_class.new(data: ft_data) }
 
   describe ".new" do
     it "creates a `FrequencyTable`" do
       expect(ft).to be_a(FrequencyTable)
     end
-  end
 
-  describe "#organizations" do
-    it "returns empty Array when initialized" do
-      expect(ft.organizations).to eq([])
+    it "accepts a Hash" do
+      expect(described_class.new(data: {})).to be_a described_class
     end
 
-    it "returns each org in the cluster" do
-      ft.add_ht_item ht1
-      ft.add_ht_item ht2
-      expect(ft.organizations).to eq([:umich, :upenn])
+    it "operates on a copy of the initializer data" do
+      ft = described_class.new(data: umich_data)
+      umich_data[:umich][:spm][:"1"] = 10
+      expect(ft.fetch(organization: :umich, format: :spm, bucket: 1)).to eq(1)
     end
-  end
 
-  describe "#[]" do
-    it "returns empty Array for unknown organization" do
-      ft.add_ht_item ht1
-      ft.add_ht_item ht2
-      expect(ft["no such organization"][:spm]).to eq({})
+    it "accepts JSON" do
+      expect(described_class.new(data: "{}")).to be_a described_class
     end
 
-    it "returns empty Array for unknown format" do
-      ft.add_ht_item ht1
-      ft.add_ht_item ht2
-      expect(ft[:umich][:no_such_format]).to eq({})
+    it "round-trips JSON" do
+      round_tripped = described_class.new(data: ft_with_data.to_json)
+      expect(round_tripped.organizations.sort).to eq [:umich, :upenn].sort
+      expect(round_tripped.fetch(organization: :umich)).to eq(ft_with_data.fetch(organization: :umich))
+      expect(round_tripped.fetch(organization: :upenn)).to eq(ft_with_data.fetch(organization: :upenn))
     end
+  end
 
-    it "returns an Array of statistics when available" do
-      ft.add_ht_item ht1
-      ft.add_ht_item ht2
-      expect(ft[:umich][:spm]).to eq({1 => 1})
-      expect(ft[:upenn][:spm]).to eq({1 => 1})
+  describe "#organizations" do
```

Comment:
Possibly drop `organizations` and these tests if `organizations` isn't used anywhere else, or move the method to the spec file as a test method.